client/status/status.js

```diff
@@ -178,20 +201,48 @@ fetchJson('config').then(config => {
   document.forms[0].config.value = JSON.stringify(
       config.current, null, '  ');
 
+  let modulesInPlaylist = buildPlayableModuleMap(config);
+
   // Draw the list of available modules in the "play immediately" section.
   let module_list = document.getElementById('module_list');
   config.current.modules.forEach(function(element) {
```

Comment:
"element" is a really terrible variable name for this in the context of dom construction...how about "module"?